packages/core/src/components/blind/blind.tsx

```diff
@@ -129,12 +125,9 @@ export class Blind {
             'blind-header': true,
             closed: this.collapsed,
           }}
-          type="button"
```

Comment:
please do not remove type='button' otherwise some browser will use the button as form submit

Reply:
added